.travis.yml

```diff
@@ -5,5 +5,7 @@ import:
 
 env:
   global:
-    # require at minimum recipe 4.7 so that redux based toast notification are available
-    - REQUIRE_RECIPE="4.7.x-dev || "
+    - REQUIRE_RECIPE="4.x-dev"
```

Comment:
It's still trying to test 4.7, 4.6 and 4.5 releases. That's preventing the builds from being :green_circle:

Reply:
I've tried changing this to use a different provision

Reply:
New provision worked, travis green